lib/seahorn/BvOpSem2.cc

```diff
@@ -555,6 +560,42 @@ class OpSemVisitor : public InstVisitor<OpSemVisitor>, OpSemVisitorBase {
     setValue(*CS.getInstruction(), res);
   }
 
+  // TODO: visit.assert.if.not is not supported
```

Comment:
move comment inside the function. Typo `visit.assert...` should be just `assert...`

Reply:
ack.